cw/cw305/cw_segmented.py

```diff
@@ -89,9 +89,15 @@ def __init__(self, num_samples=740, scope_gain=23, scope=None):
             if version.parse(cw.__version__) != version.parse("5.5.0"):
                 raise IOError("ChipWhisperer Version must be 5.5.0 (from commit 099807207f3351d16e7988d8f0cccf6d570f306a) for CW-Lite with Segmenting")
 
-        self._configure_scope(scope_gain)
+        if not self._is_husky:
+            offset = offset // 2
+        self._configure_scope(scope_gain, offset)
+
         self.num_segments = 1
-        self.num_samples = num_samples
+        if self._is_husky:
+            self.num_samples = num_samples
+        else:
+            self.num_samples = num_samples // 2
```

Comment:
I think it would be better to have different yaml files for different setups instead of updating these numbers under the hood. WDYT?

Reply:
I thought about that, too. But I think as most of us use Husky anyway it's probably a waste of effort to keep maintaining config files for CW-Lite also.

Reply:
SG, would you mind creating an issue and adding a comment here to remind us to deprecate the cw-lite support (I guess any piece of code that checks `if husky`) when we get the chance? Thanks!

Reply:
Good point, I've filed an issue here #67 .

Reply:
Thanks!